libplatsupport/src/plat/imx6/clock.c

```diff
@@ -156,17 +157,18 @@ struct ccm_alg_regs {
     uint32_t  res8[4];
     /* MISC2 */
     alg_sct_t misc2;                 /* 0x170 */
-    uint32_t  res9[11];
-    /* USB phy control - Implemented here for the sake of
-     * componentisation since it shares the same register space */
+    uint32_t  res9[8];
```

Comment:
Will reducing this to 8 change the addresses of the phy1 phy2 regs below? What was the reason for this?

Reply:
See the comment with the address. 11 is clearly wrong here. `alg_sct_t` is 16 byte. So 0x170 + 16 + 4*8 = 0x1a0. I thin the 11 came from the assumption that `alg_sct_t` is 4 bytes only- Or even from before `alg_sct_t` existed and the was simply a uint32.

Reply:
Ah cool! Good spot. Have these changes been tested?

Reply:
I'm using these changes for a few months in my QEMU NIC support branch now. However, I have not been using the USB PHY so far. This change specifically comes purely from code/datasheet reviews.